WordPress/Classes/ViewRelated/Reader/ReaderPostHeaderView.m

```diff
@@ -39,28 +50,63 @@ - (CGSize)intrinsicContentSize
     return CGSizeMake(PostHeaderViewAvatarSize, PostHeaderViewAvatarSize);
 }
 
+- (void)setShowsDisclosureIndicator:(BOOL)showsDisclosure
+{
+    _showsDisclosureIndicator = showsDisclosure;
+    [self refreshDisclosureButton];
+}
+
+- (UIImage *)avatarImage
+{
+    return self.avatarImageView.image;
+}
+
 - (void)setAvatarImage:(UIImage *)image
 {
     self.avatarImageView.image = image;
 }
 
+- (NSString *)title
+{
+    return self.titleLabel.text;
+}
+
 - (void)setTitle:(NSString *)title
 {
     self.titleLabel.text = title;
 }
 
+- (NSString *)subtitle
+{
+    return self.subtitleLabel.text;
+}
+
 - (void)setSubtitle:(NSString *)title
 {
     self.subtitleLabel.text = title;
 }
 
+
 #pragma mark - Private Methods
 
+- (void)refreshDisclosureButton
+{
+    // Note: iOS 7 doesn't allow us to simply disable a constraint. Let's improve this once the deploymentTarget is updated!
```

Comment:
Maybe add a TODO: so this will get picked up by the inspector? I think we will be less likely to forget about it that way.